src/codegen.cpp

```diff
@@ -7472,11 +7472,11 @@ extern "C" void jl_init_codegen(void)
     ;
     Triple TheTriple(sys::getProcessTriple());
 #if defined(FORCE_ELF)
-#if JL_LLVM_VERSION >= 30500
-    TheTriple.setObjectFormat(Triple::ELF);
-#else
-    TheTriple.setEnvironment(Triple::ELF);
-#endif
+    #if JL_LLVM_VERSION >= 30500
```

Comment:
made more sense with the previous alignment, we usually left-align almost all preprocessor directives - nesting can be made clearer with comments on the `#endif` lines marking which they're closing